content/research/market-health/posts/2026-05-15-pump-and-dump-anatomy/index.md

```
@@ -0,0 +1,136 @@
+
+### Trade-count burst score
+
+```
+burst_score_t = (count_t - rolling_median(count, 30s)) / rolling_mad(count, 30s)
+```
+
+Median absolute deviation (MAD) is preferred to standard deviation because the underlying series is non-Gaussian (heavy-tailed). A `burst_score_t > 30` in any 1-second bucket is the strongest single-metric P&D flag we've found.
+
+### Buy-pressure z-score
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)